pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/BaseBrokerStarter.java

```diff
@@ -420,6 +422,12 @@ public void start()
    * @param brokerAdminApplication is the application
    */
   protected void registerExtraComponents(BrokerAdminApiApplication brokerAdminApplication) {
+    brokerAdminApplication.register(new AbstractBinder() {
```

Comment:
I thought this is for override?
Shall we move this `bind(_tableCache).to(TableCache.class);` to `start()` method.

Reply:
Got it. Moved it to start() above.